File: tests/nonsmoke/functional/CompileTests/OpenMP_tests/fortran/private.f
C***************************************************
C test the simplest helloworld in OpenMP Fortan
C Liao, 12/7/2010
C***************************************************
      PROGRAM HELLO
      include 'omp_lib.h'

      INTEGER NTHREADS, TID
!$OMP PARALLEL PRIVATE(NTHREADS, TID)
      TID = OMP_GET_THREAD_NUM()
      PRINT *, 'Hello World by thread ', TID
c$omp end parallel 
      PRINT *, 'the end'
      END

File: tests/nonsmoke/functional/CompileTests/OpenMP_tests/private.c
/*
*/
#include <stdio.h>
#ifdef _OPENMP
#include <omp.h>
#endif 

float x;
int y;

int main (int argc, char * argv[])
{
#ifdef _OPENMP
   omp_set_num_threads(4);
#endif
    x=1.0;
    y=1;
#pragma omp parallel private(x)
  {
 printf("x=%f, y=%d\n",x,y);
  }

  return 0;
}

